selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/client/TestProjectServiceClient.java

```diff
@@ -148,23 +148,24 @@ public void importProject(
       throw new IOException(format("%s not a directory", sourceFolder));
     }
 
+    Path zip = Files.createTempFile("project", projectName);
     if (PLAIN_JAVA.equals(template)) {
-      Path dotClasspath = createFile(sourceFolder.resolve(".classpath"));
-      Path dotProject = Files.createFile(sourceFolder.resolve(".project"));
-      Files.deleteIfExists(dotProject);
-      Files.deleteIfExists(dotClasspath);
+      Path tmpDir = Files.createTempDirectory("TestProject");
+      Path dotClasspath = createFile(tmpDir.resolve(".classpath"));
+      Path dotProject = Files.createFile(tmpDir.resolve(".project"));
       write(
           dotProject,
           format(
                   Resources.toString(getResource("projects/jdt-ls-project-files/project"), UTF_8),
```

Comment:
Despite the fact that fixup looks good, this utility class shouldn't stick to concrete test project. So concrete project `"projects/jdt-ls-project-files"` looks odd here.